src/html5_parser/node.rs

```diff
@@ -246,32 +254,32 @@ impl Node {
 
     /// Get a constant reference to the attribute value
     /// (or None if attribute doesn't exist)
-    pub fn get_attribute(&self, name: &str) -> Result<Option<&String>, String> {
+    pub fn get_attribute(&self, name: &str) -> Option<&String> {
         if self.type_of() != NodeType::Element {
-            return Err(ATTRIBUTE_NODETYPE_ERR_MSG.into());
+            return None;
         }
 
         let mut value: Option<&String> = None;
         if let NodeData::Element { attributes, .. } = &self.data {
             value = attributes.get(name);
         }
 
-        Ok(value)
+        value
```

Comment:
We'll have to keep our eye on whether there are error cases in CSS that need to be handled.  But this change feels like a good next step.  I'm thinking we can backfill error handling when we discover that it's needed.